source/Aconcagua-Quantitative-Analysis/MultipleUnitOfMeasure.class.st

```diff
@@ -0,0 +1,116 @@
+"
+Multiple and submultiple units of measurement are used to express quantities that are larger
+or smaller than the base unit in a more convenient or practical way. These units are derived
+by applying a factor to the base unit, enabling easier representation of both very large and
+very small quantities.
+
+Multiple units are larger than the base unit and submultiple units are smaller than the base unit.
+"
+Class {
+	#name : 'MultipleUnitOfMeasure',
+	#superclass : 'UnitOfMeasure',
+	#instVars : [
+		'baseUnit',
+		'factor',
+		'names'
+	],
+	#category : 'Aconcagua-Quantitative-Analysis',
+	#package : 'Aconcagua-Quantitative-Analysis'
+}
+
+{ #category : 'instance creation' }
+MultipleUnitOfMeasure class >> basedOn: baseUnit prefixedBy: namePrefix symbolPrefixedBy: symbolPrefix factor: factor [
+
+  ^ self
+      basedOn: baseUnit
+      singularNamed: namePrefix , baseUnit singularName
+      pluralNamed: namePrefix , baseUnit pluralName
+      symbol: symbolPrefix , baseUnit symbol
+      factor: factor
+]
+
+{ #category : 'instance creation' }
+MultipleUnitOfMeasure class >> basedOn: baseUnit prefixedBy: namePrefix symbolPrefixedBy: symbolPrefix raisedTo: exponent [
+
+  ^ self
+      basedOn: baseUnit
+      prefixedBy: namePrefix
+      symbolPrefixedBy: symbolPrefix
+      factor: ( 10 raisedTo: exponent )
+]
+
+{ #category : 'instance creation' }
+MultipleUnitOfMeasure class >> basedOn: baseUnit singularNamed: singularName pluralNamed: pluralName symbol: symbol factor: factor [
```

Comment:
I'd rather use `representedBy:` instead of `symbol:`

Reply:
I'm using `symbol` because it's the name given in the ISO reference materials